authentication/forms.py

```diff
@@ -26,12 +42,14 @@ class Meta:
         model = User
         fields = ('first_name', 'last_name', 'username', 'email')
 
-
-
-
-
-
-
-
-
-
+    def email_clean(self):
```

Comment:
nit: This duplication in `SignUpForm` and `UpdateProfileForm` can be avoided via `Mixin` classes

Reply:
Mixin class has been added.